Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM jinaai/jina:2.0
+FROM jinaai/jina:2.0.8
```

Comment:
let's use 2.0.10?

Reply:
this is not going to scale, to always update the version. Needs to be addressed with the versioning system currently in discussion